scripts/define_variable.sh

```diff
@@ -19,9 +19,9 @@ if [[ $ZDS_LATEX_REPO == "" ]]; then
 fi
 
 if [[ $ZDS_JDK_VERSION == "" ]]; then
-    ZDS_JDK_VERSION="11.0.4"
+    ZDS_JDK_VERSION="11.0.14.1"
     # shellcheck disable=SC2034
-    ZDS_JDK_REV="11"
+    ZDS_JDK_REV="1"
```

Comment:
On passe de 11 à 1 ? Typo ?

Reply:
Ce n'est pas une typo ! On passe de JDK version 11.0.4 révision 11 à JDK 11.0.14.1 révision 1, les révisions étant des versions avec de très légers correctifs à ma connaissance.

Reply:
C'est souvent noté sous la forme `11.0.14.1+1` comme sur la page de *release* officielle (https://github.com/adoptium/temurin11-binaries/releases/tag/jdk-11.0.14.1%2B1) mais pour une raison que j'ignore c'est séparé en deux variables dans le script d'installation.

Reply:
Ah oui, ok. 

Juste après dans le script, cette séparation en deux variables permet d'utilisateur un séparateur différent dans le nom de l'archive téléchargée et son URL de téléchargement, mais je ne vois pas pourquoi on doit changer... À changer à l'occasion.